docs/sources/datasources/aws-cloudwatch/aws-authentication/index.md

```diff
@@ -111,6 +148,47 @@ securityContext:
   runAsGroup: 472
 ```
 
+## Use `Grafana Assume Role`
+
+{{% admonition type="note" %}}
+Grafana Assume Role is currently in [private preview]({{< relref "https://grafana.com/docs/release-life-cycle" >}}) for Grafana Cloud. To get early access this feature, reach out to Customer Support and ask for the `awsDatasourcesTempCredentials` feature toggle to be enabled on your account. It is currently only available for CloudWatch.
+{{% /admonition %}}
+
+The `Grafana Assume Role` authentication provider lets you authenticate with AWS without having to create and maintain long term AWS users or rotate their access and secret keys. Instead, you can create an IAM role that has permissions to access CloudWatch and a trust relationship with Grafana's AWS account. Grafana's AWS account then makes an STS request to AWS to create temporary credentials to access your AWS data. It makes this STS request by passing along an `externalID` that's unique per Cloud account, to ensure that Grafana Cloud users can only access their own AWS data. For more information, refer to the [AWS documentation on external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
+
+To use the `Grafana Assume Role`:
+
+1. Put in a request to Customer Support to enable`awsDatasourcesTempCredentials`.
+2. Once the feature is enabled, create a new CloudWatch data source (or update an existing one) and select `Grafana Assume Role` as an authentication provider.
+3. Create a new IAM role in the AWS console, and select **Another AWS account** as the Trusted entity.
+4. Enter the account ID of the Grafana account that has permission to assume this role 008923505280 and check the **Require external ID** box.
```

Comment:
A couple questions:
In step 3, is Trusted entity a field label?
In step 4, what is the number after the word "role"?

I've made edits based on the "Trusted entity" being a field label and the number being a mistake.

```suggestion
3. Create a new IAM role in the AWS console, and select **Another AWS account** as the **Trusted entity**.
4. Enter the account ID of the Grafana account that has permission to assume this role and check the **Require external ID** box.
```

Reply:
Yeah, I think I'd describe this as a label:
<img width="1026" alt="Screenshot 2023-09-27 at 2 55 58 PM" src="https://github.com/grafana/grafana/assets/6620164/520728ca-8d7c-421b-a92d-27d188540814">

and yeah sorry that last sentence got a bit lost from me I think. The number refers to our accountid. It will be the same for everyone which is why I put it in our documentation originally but I have removed it just to be a bit more future-proof in case we update it and forget to update the documentation. hopefully this is more clear!

Reply:
Thanks for circling back! I've made a bit of an update to step and left step 4 as is!